.teamcity/src/main/kotlin/configurations/DocsTest.kt

```diff
@@ -0,0 +1,93 @@
+package configurations
+
+import common.JvmCategory
+import common.Os
+import common.applyDefaultSettings
+import common.toCapitalized
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import model.CIBuildModel
+import model.Stage
+import model.prepareTestClassesStep
+
+class DocsTestProject(
+    model: CIBuildModel,
+    stage: Stage,
+    os: Os,
+    testJava: JvmCategory,
+    configCacheEnabled: Boolean
+) : Project({
+    id("${model.projectId}_DocsTest_${testJava.version.name.toCapitalized()}_${os.asName()}")
+    name = "Docs Test - ${testJava.version.name.toCapitalized()} ${os.asName()}"
+}) {
+    val docsTests: List<BaseGradleBuildType>
+
+    init {
+        val configCacheVariants = if (configCacheEnabled) listOf(true, false) else listOf(false)
```

Comment:
🤔 I think instead of using here booleans for include/exclude and with/without config cache it would be better to use enums so it is clearer what is what.